### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by aizjicod**
In this branch, I worked on the desktop version while trying to fix the responsive design of the page.

- first I worked on the header and headline and added a new background.
- after finishing that I worked on the about me section while changing the background.
- then decided to work on the last section, hence the form contact.
- for last I decided to leave the most complicated section, the work section. since it has a lot of elements and I worked with grid this time.
- after finishing the responsiveness of the page I fixed the linters of the page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,108 +50,117 @@ <h2>I’am a Software Developer</h2>
...
+                <h2>Multi-Post Stories</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <div class="tags">
+                    <a><div class="tag-btn">css</div></a>
+                    <a><div class="tag-btn">html</div></a>
+                    <a><div class="tag-btn">bootstrap</div></a>
+                    <a><div class="tag-btn">Ruby</div></a>
+                </div>
+                <div class="works__action-btn">
+                    <button type="button" class="works-btn">See Project</button>
```
Comment (elmar8287) [inline index.html:68]: - [ ] Please, consider fixing the styling for all buttons on your page. The `active` status (pressed status) for buttons is not correct. Have a look to this gif:

> The original design for buttons, considering the Figma is:
Screen Shot 2022-06-02 at 11 51 58

File: index.html [hunk 2]
```diff
@@ -203,10 +230,9 @@ <h3>Skills</h3>
...
                     <li class="twitter-icon"><a href="#"><img src="img/twitter-contact.png" alt="this is an icon for twitter" class="media-icon"></a></li>
                 </ul>
-               <div class="bottom-line">
+               <div class="bottom-line mobile-v">
                    <img src="./img/bottom-line-contact.png" alt="A line that indicates bottom of the page">
                </div>
-
             </article>
         </section>
     </main>
```
Comment (elmar8287) [inline index.html:238]: - [ ] Please, consider also fixing the `Get in touch` button. It should be left-aligned, considering the original design 👍 
Screen Shot 2022-06-02 at 11 56 59
> Figma design
Screen Shot 2022-06-02 at 11 56 48

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- The nice job is done, by implementing the desktop version
- No linter issues

## Required Changes ♻️

- [ ] Check the comments under the review.
- [ ] Please, consider fixing the Live demo and MIT.md file links in README file - they are not active 

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (sja-thedude) [reply index.html:238]: Implemented 👍

File: index.html (hunk 1 above)
Comment (sja-thedude) [reply index.html:68]: Implemented 👍

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @aizjicod, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**